react/features/base/session/constants.js

```diff
@@ -0,0 +1,12 @@
+
+export const SESSION_CONFIGURED = Symbol('SESSION_CONFIGURED');
```

Comment:
initialized?

Reply:
It's an opinion, so let's also wait for @lyubomir. I prefer CONFIGURED because it's more precise on what has happened.